common/model-views.h

```diff
@@ -30,6 +30,7 @@
 #include "cah-model.h"
 #include "../common/utilities/time/periodic_timer.h"
 #include "reflectivity/reflectivity.h"
+#include "utilities/number/stabilized-value.h"
```

Comment:
I think it should use `<>`? Same for previous line... and the one before...

Reply:
I use a concept of using <> for external headers (STL, 3-RD party) and "" for internal headers

For #include <filename> the preprocessor searches in an implementation dependent manner, normally in search directories pre-designated by the compiler/IDE. This method is normally used to include standard library header files.

For #include "filename" the preprocessor searches first in the same directory as the file containing the directive, and then follows the search path used for the #include <filename> form. This method is normally used to include programmer-defined header files.

So what is your concept?

Reply:
👍